microservices-connector/config/manager/gmc-manager.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       containers:
       - command:
         - /manager
-        image: zhlsunshine/gmcmanager:latest
+        image: zhlsunshine/gmcmanager:oneclick
```

Comment:
I see in the other PR, the docker image's version is latest, is this affirmative using "oneClick"?